packages/shared/src/schemas.ts

```diff
@@ -147,4 +147,8 @@ export type BoostedQuoteDetails = QuoteDetails & { estimatedBoostFeeBps: number
 
 export type QuoteQueryResponse = QuoteDetails & {
   boostQuote?: BoostedQuoteDetails;
+  dcaParams?: {
+    numberOfChunks: number;
+    chunkInterval: number;
```

Comment:
```suggestion
    chunkIntervalBlocks: number;
```

Reply:
actually nvm, since the FE passes this response to the openDC method lets keep it  so no transformations are necessary.